### PR TITLE
Remove magic numbers and Americanisms

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/openrct2/ride/ride.h [hunk 1]
```diff
@@ -759,7 +759,7 @@ enum {
 	RIDE_TYPE_FLAG_HAS_DATA_LOGGING = 1 << 9,
 	RIDE_TYPE_FLAG_HAS_DROPS = 1 << 10,
 	RIDE_TYPE_FLAG_NO_TEST_MODE = 1 << 11,
-	RIDE_TYPE_FLAG_TRACK_ELEMENTS_HAVE_TWO_VARIETIES = 1 << 12,	// used by rides with two varieties, like the u and o shapes of the dinghy slide and the dry and submerged track of the water coaster
+	RIDE_TYPE_FLAG_TRACK_ELEMENTS_HAVE_TWO_VARIETIES = 1 << 12,	// used by rides with two variaties, like the u and o shapes of the dinghy slide and the dry and submerged track of the water coaster
```
Comment (rwjuk) [inline src/openrct2/ride/ride.h]: "varieties" is the correct spelling

Comment (Gymnasiast) [inline src/openrct2/ride/ride.h]: That was accidental. I'll revert it.

**Comment by IntelOrca** [comment]
Any instances of *"whoa belly"* can be changed too.

**Comment by Gymnasiast** [comment]
@IntelOrca Done!

**Pull request merged**